torch/csrc/distributed/c10d/Ops.cpp

```diff
@@ -5,9 +5,9 @@
 
 namespace c10d {
 namespace {
-c10::intrusive_ptr<ProcessGroup::Work> broadcast(
-    const c10::intrusive_ptr<ProcessGroup>& process_group,
+c10::intrusive_ptr<ProcessGroup::Work> broadcast_(
```

Comment:
curious, what's the reason for adding a trailing underscore?

Reply:
It's a way to signal this is an in-place op. I guess that's almost true for every c10d ops.